api/app/reviews/tests.py

```diff
@@ -1285,3 +1288,246 @@ def test_total_number_of_pages_when_zero(self):
 
         self.assertEqual(data['total_pages'], 0)
     
+
+class ReferenceReviewRequest(ApiTestCase):
+    def static_seed_data(self):
+        # User, country and organisation is set up by ApiTestCase
+        self.first_user = self.add_user('firstuser@mail.com', 'First', 'User', 'Mx')
+
+        event = self.add_event()
+
+        reviewer1 = AppUser('r1@r.com', 'reviewer', '1', 'Mr', password='abc', organisation_id=1, )
+        candidate1 = AppUser('c1@c.com', 'candidate', '1', 'Mr', password='abc', organisation_id=1, )
+        system_admin = AppUser('sa@sa.com', 'system_admin', '1', 'Ms', password='abc', organisation_id=1, is_admin=True)
+        event_admin = AppUser('ea@ea.com', 'event_admin', '1', 'Ms', password='abc', organisation_id=1)
+        users = [reviewer1, candidate1, system_admin, event_admin]
+        for user in users:
+            user.verify()
+        db.session.add_all(users)
+        db.session.commit()

```

Comment:
Please use `self.add_user` and/or `self.add_n_users` from testing.py instead of this.

Reply:
I'm resolving this comment, because it has been updated, but not confirmed. The new comment can be found here.